### DISCUSSION TITLE
Raven DB book for 5.x planned?

### DISCUSSION THREAD

**Discussion by JezhikLaas** [ravendb/ravendb#15148] [General]
Hello,
I really enjoyed the book for version 4, is there a plan for an updated version?

Thanks, Uwe

**Comment by ayende** [comment]
There is a new RavenDB coming shortly that will cover 5.x as well: 
https://www.amazon.com/Introducing-RavenDB-Database-Modern-Persistence/dp/1484289188/ref=sr_1_1

The `Inside RavenDB` book is going to be updated after the 6.0 release.
